internal/handlers/tigris/msg_collstats.go

```diff
@@ -17,10 +17,12 @@ package tigris
 import (
 	"context"
 
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgCollStats implements HandlerInterface.
 func (h *Handler) MsgCollStats(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	return nil, errNotImplemented
+	// TODO
```

Comment:
Here and after: 
Will you add an issue link? The name of the PR shows that it's expected to implement everything related.